.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 sudo: required  # See http://docs.travis-ci.com/user/trusty-ci-environment/
 dist: trusty
 node_js:
-  - "4"
+  - "6"
```

Comment:
This is now blocking #10776 due to `fetch-mock` being incompatible with node 4. See https://travis-ci.org/ampproject/amphtml/jobs/260683377

Using node 6 for testing on Travis solves these issues.